client/src/utils/GetMeetingLink.ts

```diff
@@ -75,6 +75,17 @@ export const isValidTeamsLink = (teamsMeetingLink: string): boolean => {
   return true;
 };
 
+export const isValidRoomsLink = (roomsMeetingLink: string): boolean => {
+  if (
+    !roomsMeetingLink.startsWith(window.location.origin + '/visit') ||
+    !roomsMeetingLink.includes('roomId') ||
+    !roomsMeetingLink.includes('userId')
```

Comment:
We can be more precise here by using a regex.

For instance, just using `.includes()` here would still render this a valid rooms link: `https://localhost:3000/visit_roomId-%^&userId`

Reply:
done!